components/Common/OramaSearchBar/index.module.scss

```diff
@@ -0,0 +1,24 @@
+.oramaSearchBarFooter {
```

Comment:
The style might need some changes for dark mode

<img width="467" alt="image" src="https://github.com/nodejs/nodejs.org/assets/12037269/1a2df164-a60f-43de-bf8b-e292f31644f7">

Reply:
Solved. I was using `theme` instead of `resolvedTheme`.